### ISSUE TITLE
fonts not found where I run dev

### ISSUE THREAD

**Issue by cmzz** [JeffreyWay/laravel-mix#335]
`npm run dev`  can work with 0.5.0 but didn't  work with 0.6.0 

✗ npm run dev

    > @ dev /Users/zyxcba/www/taotaoe
    > node node_modules/cross-env/bin/cross-env.js NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js

    fallbackLoader option has been deprecated - replace with "fallback"
    loader option has been deprecated - replace with "use"
     95% emitting
     ERROR  Failed to compile with 9 errors

    These dependencies were not found in node_modules:

    * ../fonts/bootstrap/glyphicons-halflings-regular.eot
    * ../fonts/bootstrap/glyphicons-halflings-regular.woff2
    * ../fonts/bootstrap/glyphicons-halflings-regular.woff
    * ../fonts/bootstrap/glyphicons-halflings-regular.ttf
    * ../fonts/bootstrap/glyphicons-halflings-regular.svg
    * ../fonts/iconfont/iconfont.eot
    * ../fonts/iconfont/iconfont.woff
    * ../fonts/iconfont/iconfont.ttf
    * ../fonts/iconfont/iconfont.svg

    Did you forget to run npm install --save for them?
                              Asset      Size  Chunks                    Chunk Names
    /js/app.b9d8f43838d61cf2c090.js   1.27 MB       0  [emitted]  [big]  /js/app
                  mix-manifest.json  53 bytes          [emitted]

    npm ERR! Darwin 16.4.0
    npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "dev"
    npm ERR! node v7.5.0
    npm ERR! npm  v4.1.2
    npm ERR! code ELIFECYCLE
    npm ERR! @ dev: `node node_modules/cross-env/bin/cross-env.js NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js`
    npm ERR! Exit status 2
    npm ERR!
    npm ERR! Failed at the @ dev script 'node node_modules/cross-env/bin/cross-env.js NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js'.
    npm ERR! Make sure you have the latest version of node.js and npm installed.
    npm ERR! If you do, this is most likely a problem with the  package,
    npm ERR! not with npm itself.
    npm ERR! Tell the author that this fails on your system:
    npm ERR!     node node_modules/cross-env/bin/cross-env.js NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js
    npm ERR! You can get information on how to open an issue for this project with:
    npm ERR!     npm bugs
    npm ERR! Or if that isn't available, you can get their info via:
    npm ERR!     npm owner ls
    npm ERR! There is likely additional logging output above.

All font files exites: 
    ✗ ls public/fonts/bootstrap/
    glyphicons-halflings-regular.eot   glyphicons-halflings-regular.ttf   glyphicons-halflings-regular.woff2
    glyphicons-halflings-regular.svg   glyphicons-halflings-regular.woff

My webpack.mix.js file:

mix
    .js('resources/assets/js/app.js', 'public/js')
    .sass('resources/assets/sass/app.scss', 'public/css')
    .version()

**Comment by JeffreyWay** [comment]
Please add `$icon-font-path` to your `variables.scss` file, like this: 

https://github.com/laravel/laravel/commit/2cc5f349d495991447beb08cf04f5f607e95fbb6

**Comment by jevets** [comment]
I just ran into something similar but w/ font-awesome.

I fixed easily by:

1. Setting up a mix.copy() in webpack.mix.js
2. Setting the font-awesome path prior to importing the .scss file in my main app.scss

```js
// webpack.mix.js
mix.copy('node_modules/font-awesome/fonts', 'public/fonts');
```

```scss
// app.scss
$fa-font-path: '/fonts'
@import 'node_modules/font-awesome/scss/font-awesome.scss'
```

If you're `version()`ing, you'll probably need to copy fonts to `public/build/fonts`.

Something similar should work for bootstrap, probably setting `$icon-font-path`, see https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_glyphicons.scss#L14.
